javascript/code-challenges/challenges-02.test.js

```diff
@@ -0,0 +1,307 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+
+CHALLENGE 1 - Review
+
+Write a function named raisedToTheThird that takes in an array of numbers and returns a new array of each of those numbers raised to the 3rd power (hint: look up Math.pow()). Use forEach to solve this problem.
+
+------------------------------------------------------------------------------------------------ */
+const raise = [];
+
+const raisedToTheThird = (arr) => {
+  arr.forEach(value => raise.push(Math.pow(value, 3)));
+  return raise;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named addOne that, given an array of numbers, uses map to return a new array with each value simply incremented by 1.
+------------------------------------------------------------------------------------------------ */
+
+const addValue = [];
+const addOne = (arr) => {
+  arr.map(value => addValue.push(value + 1));
+  return addValue;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 3
+
+Write a function named addQuestion that, given an array of strings, uses map to return a new array containing each string followed by a question mark character.
+------------------------------------------------------------------------------------------------ */
+const Qmark = [];
```

Comment:
Don't create variables outside of your functions. You do not want any global variables in your code challenges.